### PR TITLE
#214 new attributes `failed_builds` and `succeeded_builds` for `pull-was-merged`

### LINKED ISSUE #214: new attributes `failed_builds` and `succeeded_builds` for `pull-was-merged`
When we create `pull-was-merged` in `github-events.rb`, let's ask GitHub API for the number of succeeded and failed workflow builds in this pull request. Let's save this number as `failed_builds` and `succeeded_builds` attributes into the fact.

### PR DESCRIPTION
**Pull request by Suban05**
@yegor256 merge https://github.com/zerocracy/fbe/pull/77 PR before

Closes #214

### CODE REVIEW

**Comment by Suban05** [comment]
@Yegorov @tank_bohr please review
tests fails because this pr needs this pr https://github.com/zerocracy/fbe/pull/77

File: test/judges/test-github-events.rb [hunk 1]
```diff
@@ -641,6 +636,14 @@ def test_pull_request_event_with_comments
     assert_equal(1, f.comments_resolved)
   end
 
+  def test_count_numbers_of_workflow_builds
+    fb = Factbase.new
+    load_it('github-events', fb, Judges::Options.new({ 'repositories' => 'foo/foo', 'testing' => true }))
```
Comment (Yegorov) [inline test/judges/test-github-events.rb:641]: @Suban05 Why not use web mock for this cases? Testing mode used for run yml files, for example - https://github.com/zerocracy/judges-action/blob/master/judges/github-events/quick-scan.yml

**Review by Yegorov — COMMENTED** [review]
@Suban05 Look good, but there is some question.

**Review by tank-bohr — APPROVED** [review]
:+1:

File: test/judges/test-github-events.rb (hunk 1 above)
Comment (Suban05) [reply test/judges/test-github-events.rb:641]: @Yegorov good question. For the `pull-was-merged` event,  we use `github_graph`. Maybe I don't understand something, but it's very difficult to mock this thing by `WebMock`. And I use a fake client for `Fbe.github_graph`. However, there is an `options` argument  in both clients (`Fbe.octo` and `Fbe.github_graph`), and when I set `testing` globally, `Fbe.octo` also becomes a fake client. That's why I use fake clients

**Comment by Suban05** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@Suban05 can't merge, there are git conflicts, please resolve them

**Comment by Suban05** [comment]
@yegor256 done

**Comment by yegor256** [comment]
@Suban05 thanks!

**Comment by 0crat** [comment]
@Yegorov Thanks for your code review! Here's how your +4 points were calculated: +8 base points, -24 for self-review, +16 for hits-of-code (capped), -10 for few comments (2), +6 for minimum reward. We value your contributions! Your balance is now [+435](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Keep up the good work! 💻👍

**Comment by 0crat** [comment]
@tank-bohr Hey there! 👋 Great job on the review! You've snagged +4 points for this one. Here's the breakdown: +8 as the base, -10 because we only saw 2 comments (we love detailed feedback!), and +6 to keep you above the minimum. Remember, more comments could boost your score next time - up to 8 points for chatty reviews! Your [running balance](https://zerocracy.github.io/judges-action/zerocracy-vitals.html) is now at +136. Keep up the good work and happy coding! 💻🚀

**Comment by 0crat** [comment]
@Suban05 Great job on your contribution! 🎉 You've earned +21 points: +16 base, +5 for 53 [hits-of-code](https://www.yegor256.com/2014/11/14/hits-of-code.html). No deductions for excessive code or lack of reviews. Your balance is now [+456](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Keep up the good work and remember, [quality matters](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) in our review process. Looking forward to your next contribution!

**Pull request merged**